random.cabal

```diff
@@ -60,3 +58,12 @@ Test-Suite TestRandomIOs
     hs-source-dirs: tests
     build-depends:  base >= 3 && < 5, random
     ghc-options:    -rtsopts -O2 -with-rtsopts=-M1M
+
+Test-Suite doctests
+  type:             exitcode-stdio-1.0
+  hs-source-dirs:   tests
+  main-is:          doctests.hs
+  build-depends:    base
+                  , doctest >=0.15
+                  , random
+  default-language: Haskell2010
```

Comment:
Nit: git hates files that don't end in a newline - please add one.

https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline